data/adopters.yml

```diff
@@ -447,4 +447,8 @@ adopters:
     - name: F1Tenth Autonomous Racing Community
       homepage_url: https://f1tenth.org/
       logo: logo-f1tenth.png
-      logo_white: logo-f1tenth-white.png
\ No newline at end of file
+      logo_white: logo-f1tenth-white.png
+    - name: Bosch.IO
+      homepage_url: https://www.bosch-iot-suite.com/service/hub/
+      logo: logo-bosch.svg
```

Comment:
Did you forget to add the logos or are they already in the repo?

Reply:
Already in, we have Bosch in other projects.

Reply:
it's here https://github.com/flora8984461/iot.eclipse.org/blob/yiliu/master/addBosch/static/assets/images/adopters/logo-bosch.svg 
and the white https://github.com/flora8984461/iot.eclipse.org/blob/yiliu/master/addBosch/static/assets/images/adopters/logo-bosch-white.svg